test/verify/check-networking-firewall

```diff
@@ -161,6 +169,12 @@ class TestFirewall(NetworkCase):
         m.execute("firewall-cmd --permanent --new-service=empty && firewall-cmd --reload && firewall-cmd --add-service=empty")
         b.wait_present(".zone-section[data-id='public'] tr[data-row-id='empty']")
 
+        if m.image not in ["rhel-8-2-distropkg"]: # Changed in #13686
+            m.execute("firewall-cmd --add-port=9999/udp")
+            b.wait_in_text(".zone-section[data-id='public'] tr[data-row-id='public-ports']", "9999")
+            m.execute("firewall-cmd --remove-port=9999/udp")
+            b.wait_not_present(".zone-section[data-id='public'] tr[data-row-id='public-ports']")
```

Comment:
Can you please test a TCP port as well, and a second port to make sure that the formatting and sorting works as intended?

Reply:
sure